components/dashboard/src/Menu.tsx

```diff
@@ -97,9 +97,12 @@ export default function Menu() {
     // Hide most of the top menu when in a full-page form.
     const isMinimalUI = inResource(location.pathname, ["new", "teams/new", "open"]);
     const isWorkspacesUI = inResource(location.pathname, ["workspaces"]);
-    const isAccountUI = inResource(location.pathname, [
+    const isPersonalSettingsUI = inResource(location.pathname, [
```

Comment:
[random] `inResource` is super misleading. I expected this to check if the pathname contains that resource, but it's actually checking if it starts with those resources.

Reply:
Thanks for calling this out @easyCZ! I must admit I only briefly skimmed this code while implementing my fix, but I agree about the name `inResource` being super unclear and unhelpful. Do you have a better suggestion in mind? 🙂

Reply:
`resourceStartsWith` or `locationStartsWith` comes to mind. Mostly to communicate it's not just a substring match, but a starts with match